packages/dev/docs/pages/internationalized/index.mdx

```diff
@@ -0,0 +1,64 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import {Highlights, Hero} from '@react-spectrum/docs';
+import heroNarrow from 'url:../assets/internationalized_mobile@2x.png?width=976';
+import heroNarrow2x from 'url:../assets/internationalized_mobile@2x.png?width=1952';
+import heroNarrowWebp from 'url:../assets/internationalized_mobile@2x.png?as=webp&width=976';
+import heroNarrow2xWebp from 'url:../assets/internationalized_mobile@2x.png?as=webp&width=1952';
+import heroWide from 'url:../assets/internationalized@2x.png?width=976';
+import heroWide2x from 'url:../assets/internationalized@2x.png?width=1952';
+import heroWideWebp from 'url:../assets/internationalized@2x.png?as=webp&width=976';
+import heroWide2xWebp from 'url:../assets/internationalized@2x.png?as=webp&width=1952';
+import clsx from 'clsx';
+import linkStyle from '@adobe/spectrum-css-temp/components/link/vars.css';
+import styles from '@react-spectrum/docs/src/docs.css';
+import typographyStyles from '@adobe/spectrum-css-temp/components/typography/vars.css';
+
+---
+category: Libraries
+description: Internationalization utilities
+---
```

Comment:
We will need some way to prevent this page from going out if we merge it before we are ready for the release.